samples/bluetooth/ble_radio_notification/prj.conf

```
@@ -0,0 +1,18 @@
+CONFIG_LOG=y
+CONFIG_LOG_BACKEND_BM_UARTE=y
+
+CONFIG_SOFTDEVICE=y
+CONFIG_NRF_SDH=y
+
+CONFIG_BLE_ADV=y
+CONFIG_BLE_ADV_NAME="nRF_BM_HRS"
+CONFIG_BLE_ADV_EXTENDED_ADVERTISING=n
+CONFIG_BLE_ADV_DIRECTED_ADVERTISING=n
+
```

Perhaps:
```suggestion
CONFIG_BLE_ADV_FAST_ADVERTISING_INTERVAL=400
CONFIG_BLE_ADV_SLOW_ADVERTISING_INTERVAL=1600
```
So it blinks at 4Hz, then 1 Hz. The default blinks a bit fast.